site/content/docs/04-compiler-and-api/01-svelte-compiler.md

```diff
@@ -238,8 +238,8 @@ The `script` and `style` functions receive the contents of `<script>` and `<styl
 If a `dependencies` array is returned, it will be included in the result object. This is used by packages like [rollup-plugin-svelte](https://github.com/sveltejs/rollup-plugin-svelte) to watch additional files for changes, in the case where your `<style>` tag has an `@import` (for example).
 
 ```js
-const svelte = require('svelte/compiler');
-const sass = require('node-sass');
+import svelte from 'svelte/compiler';
+import sass from 'node-sass';
```

Comment:
Should we maybe update this to use dart-sass? Recommending something as obnoxious as node-sass can't be a good impression

Reply:
that's what `svelte-preprocess` does, so I guess it wouldn't hurt to change for consistency: https://github.com/sveltejs/svelte-preprocess/blob/537b97536c2a48574ca748dccf3fcc0dc6d3ffc5/src/transformers/scss.ts

Reply:
This uses a deprecated API of `sass`, so I tested on staclkblitz with new `compileStringAsync` API, works, so gonna keep that here https://stackblitz.com/edit/node-ncrbch?file=index.js

Reply:
maybe you should send it as a PR to `svelte-preprocess` as well :smile: